src/cli/cli.ts

```diff
@@ -67,6 +67,14 @@ function getSettingWithDefaultValue<TValue>(settingName: string, defaultValue: T
   }
 }
 
+function getClientId(): string | undefined {
+  return cli.getSettingWithDefaultValue(settingsNames.clientId, process.env.CLIMICROSOFT365_ENTRAAPPID || process.env.CLIMICROSOFT365_AADAPPID);
```

Comment:
Hi @waldekmastykarz, I notice you are saving the `clientId` and `tenantId` of the app registration to the m365 config. Isn't that a bit problematic as it's not going to work soundly with our multi-account setup?

Reply:
Why wouldn't it work with our multiaccount setup? It's not much different from what we had previously, where the app ID and tenant ID were included in our code, now they're in the config. We still offer the ability to store that information per connection and restore it from there. That functionality remains intact.

Reply:
Well, if you would execute the setup command on multiple tenants, it would only remember the last ClientId/tenant, right? So it's tied to one tenant, and yet it will be used as default tenant Id / default app Id for new connections. I guess it's not a real problem, but it will just take some getting used to. 

For example: if I do this:
```powershell
# Create app reg on blimped tenant
m365 setup 

# Sign into blimped tenant
m365 login --connectionName "blimped"

# Create app reg on contoso tenant
m365 setup  

# Sign into contoso tenant
m365 login --connectionName "contoso"

# The above will work. But if I know try to relogin to blimped
m365 login --connectionName "blimped"

# ...I'll get an exception in the login flow, because it defaulted to the App from Contoso.
![image](https://github.com/user-attachments/assets/fc92f28b-09ea-477c-a706-bfdd768b9a88)
```

You might say I could have used "m365 connection use" to switch to blimped. And that's true. But for some customers of mine, refresh tokens expire every day, so I have to refresh the connection each day, using m365 login. 

Maybe it's just something to getting used to....it's a drawback...

Reply:
I wonder, should we save the tenantName somewhere as well, and show the appId and tenantName in a log line used in the login process if no tenant-specific settings were used as options or env variables.

for example: 
![image](https://github.com/user-attachments/assets/2206b126-925b-48dd-b29a-e413afd0f626)

Reply:
I wonder if we're not making it too complicated by mixing setup and multiaccount setup. To me, setup is meant to help you setup CLI for initial use, to create or configure a default app. I don't people run setup each time they connect to a different tenant. With multiaccount setup, the app information is stored in the connection and you likely already have an Entra app in the target tenant, so there's really no need to use setup. I suggest that we keep setup and multiaccount use separate.